modules/api-svc/package.json

```diff
@@ -62,58 +62,58 @@
     "url": "git@github.com:mojaloop/sdk-scheme-adapter.git"
   },
   "dependencies": {
-    "@koa/cors": "^4.0.0",
-    "@mojaloop/api-snippets": "v17.3.0-snapshot.0",
+    "@koa/cors": "^5.0.0",
+    "@mojaloop/api-snippets": "v17.4.0",
     "@mojaloop/central-services-error-handling": "^12.0.7",
     "@mojaloop/central-services-logger": "^11.2.2",
     "@mojaloop/central-services-metrics": "^12.0.8",
     "@mojaloop/central-services-shared": "v18.2.0-snapshot.16",
     "@mojaloop/event-sdk": "^14.0.0",
     "@mojaloop/sdk-scheme-adapter-private-shared-lib": "workspace:^",
-    "@mojaloop/sdk-standard-components": "v17.2.0-snapshot.4",
+    "@mojaloop/sdk-standard-components": "v17.4.0",
```

Comment:
this version has an issue with `jwsSigner.sign()`, just in case:
https://infitx-technologies.atlassian.net/browse/IPROD-427

Reply:
Thanks @geka-evk. Dependency was reverted and locked until fixed.